modules/ROOT/pages/supported-types-of-tests.adoc

```diff
@@ -0,0 +1,41 @@
+= MUnit Supported Types of Tests
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following are definitions and concepts that MuleSoft recommends to take into account while designing and organizing test cases for your Mule applications. Understanding these definitions and concepts will help you build more robust MUnit tests by covering multiple edge cases, automate your tests through your CI/CD pipeline, and maximize the quality coverage for your Mule application.
```

Comment:
GLOBALLY avoid introducing topics directly by referring to "the following text/section/definitions/concepts...."
Get straight to the point by summarizing the content of the topic set, addressing the audience directly as "you" and telling "you" why they want or need this information, to achieve a specific user goal:
```suggestion
Before you design and organize test cases for your Mule applications, you should understand the types of tests supported by MUnit: unit testing, integration testing, and functional testing. Having this knowledge can help you to build more robust MUnit tests by covering multiple edge cases, to automate your tests through your CI/CD pipeline, and to maximize the quality coverage for your Mule application.
```